### ISSUE TITLE
Get rid of microcircuit multimeters if they are not needed

### ISSUE THREAD

**Issue by espenhgn** [INM-6/hybridLFPy#22]
In particular `depth_res_input_spikes*.dat` results in a stupid amount of files (per cell type and per virtual process) that nothing gets recorded to by default. 

If
```
self.n_rec_voltage = 0
self.n_rec_input_spikes = 0
self.n_rec_depth_resolved_input = 0
```
no such files should be created.

**Comment by espenhgn** [comment]
Fixed in PR #9

**Comment by espenhgn** [comment]
Fixed by merging #9
